ts/private/ts_project.bzl

```diff
@@ -176,12 +178,12 @@ This is an error because Bazel does not run actions unless their outputs are nee
             run_action(
                 ctx,
                 link_workspace_root = ctx.attr.link_workspace_root,
-                executable = "tsc",
+                executable = executable,
```

Comment:
oh this needs some cleanup, we still have the code from rules_nodejs which is no longer applicable based on rules_js.

Reply:
Thanks for doing this 😊